README.rst

```diff
@@ -49,19 +49,22 @@ Features
 Installation
 ------------
 
-*rst2pdf* supports Python 3.8 or greater. Version 0.99 was the last version to support Python 3.6 & 3.7, with 0.97 the last version to support Python 2.7.
+*rst2pdf* supports Python 3.8 or greater. Version 0.99 was the last version to support Python 3.6 & 3.7, with 0.97 the
+last version to support Python 2.7.
 
 Install from PyPI
 ~~~~~~~~~~~~~~~~~
 
-The latest released version may be installed from PyPI by using ``pipx``::
+The latest released version may be installed from PyPI by using `uv <https://docs.astral.sh/uv>`_::
 
-    $ pipx install rst2pdf
+    $ uv tool install rst2pdf
```

Comment:
Here `uv tool` is a replacement for `pipx`, but I note the absence of `pip` instructions for downstream users who may not be using `pipx` or `uv` in their own workflows.

Would it make sense to add `pip` instructions below as another alternative method?

Reply:
We removed ` pip` in favour of `pip`  due PEP 668's which impacted Linux users (#1165). 

As rst2pdf is targeted at end-users, we should provide pipx in addition to uv.

Reply:
Ah that makes sense, moving away from `pip` as an installation method for system wide packages.

But now `uv tool install rst2pdf` is equivalent to `pipx install rst2pdf` as they both create and manage isolated environments internally. The only difference being `uv` is faster.

Not sure if there's any benefit to having both `pipx` and `uv` instructions here or if it would only confuse users?

Reply:
I think `uv` is too new to be the only one. I can imagine developers moving to `uv` sooner than consumers.

Reply:
No worries, the original `pipx` instructions are back in now with `uv` being an additional method and not the only suggested one.

Reply:
Thanks! Having both uv and pipx sounds like the right current approach to me. As a user of many different tech stacks, I don't always want to be using the latest and greatest of all their dependency management tools!